config/conf.json

```diff
@@ -26,7 +26,7 @@
                     "tidyverse"
                 ]
             },
-            "version" : "2.1.11",
+            "version" : "2.2.0",
             "automated_flags" : {
```

Comment:
did a minor version bump for every image, let me know if you think this deserves a major bump?

Reply:
I am good with the minor bump 👍